doc/data/messages/t/too-few-function-args/bad.py

```diff
@@ -0,0 +1,2 @@
+# As of 2024-08-13, `too-few-function-args` is only implemented for `isinstance`
```

Comment:
The comment could be moved to details.rst in the same folder, and then it will appear in the docs in a more logical way.

Reply:
Added it to `details.rst` in [2d6db8a](https://github.com/pylint-dev/pylint/pull/9868/commits/2d6db8a273fab85e9d50c66eed8527598759ca50)